components/push/ffi/src/lib.rs

```diff
@@ -51,7 +51,7 @@ pub extern "C" fn push_connection_new(
 ) -> u64 {
     MANAGER.insert_with_result(error, || {
         log::debug!(
```

Comment:
This was already here but just to check, is any of this PII? If so we should log::trace it instead

Reply:
Push goes out of its way to avoid PII, but still good to sanity check.

The Sender_ID is service specific (for FCM/APNS/etc.), and the Registration_ID is application specific (for connections to Firefox). Both are readily available on the device, and within the application. There's no direct harm in this going into the log, since far more significant PII could be attained easier. I'm also totally fine with making this a trace if you feel that there's even the slightest risk.